deployments/goerli/usdc/roots.json

```diff
@@ -8,6 +8,5 @@
     "comet": "0x44Ac0af6F39c98d59Ed6D16404210656dB912Ffd",
     "configurator": "0xdE0e9466F406Eb0922746861Bc09785bED912C80",
     "rewards": "0xef9e070044d62C38D2e316146dDe92AD02CF2c2c",
-    "bulker": "0x2677D9ca7477A81A394C324B97F3D7907C6727Fb",
-    "fxRoot": "0x3d1d3e34f7fb6d26245e6640e1c50710efff15ba"
```

Comment:
If `deploy` is run again, it will add `fxRoot` back into `roots.json` right?

Reply:
yep, it should